numba/tests/test_range.py

```diff
@@ -177,6 +187,31 @@ def test_range_attrs(self):
         for arg in arglist:
             self.assertEqual(cfunc(*arg), pyfunc(*arg))
 
+    @tag('important')
+    def test_range_contains(self):
+        pyfunc = range_contains
+        arglist = [(0, 0, 1),
+                   (0, -1, 1),
+                   (-1, 1, 1),
+                   (-1, 4, 1),
+                   (-1, 4, 10),
+                   (5, -5, -2),]
+
+        vallist = [-10, -6, -5, -4, -2, -1, 0,
```

Comment:
It would be good to split the `vallist` into several for each type to make it easier to read. You can then chain them together. This will make it clearer what values are being tested in what type category.

```
bool_vals = ...
int_vals = ...
float_vals = ...
complex_vals = ...
non_numeric_vals = ...
for val in int_vals + float_vals ....:
    pass
```

Reply:
Done